src/resource-rental/resource-rental.ts

```diff
@@ -53,7 +53,7 @@ export class ResourceRental {
   ) {
     this.networkNode = this.resourceRentalOptions?.networkNode;
 
-    this.createExeUnit(this.abortController.signal);
```

Comment:
I've read the explanation which you provided on the PR. Still, I'd recommend gracefully logging the error on `debug` level for the sake of troubleshooting in future.